koku/common/sql_retries.py

```diff
@@ -0,0 +1,72 @@
+import functools
+import logging
+import random
+import time
+import typing as t
+
+LOG = logging.getLogger(__name__)
+
+
+class TrinoQueryError(Exception):
+    pass
+
+
+def log_json(msg, log_ref, context):
+    return f"{msg} - {log_ref} - {context}"
+
+
+def retry(
+    original_callable: t.Callable = None,
+    *,
+    retries: int = 3,
+    retry_on: tuple[Exception, ...],
```

Comment:
This actually needs to be a subclass of Exception.
```suggestion
    retry_on: tuple[t.Type[Exception], ...],
```

Reply:
Set the default to just retry on any excption.
```suggestion
    retry_on: tuple[Exception, ...] = (Exception, ),
```